pages/catalog/CatalogBookHandler.inc.php

```diff
@@ -39,8 +39,12 @@ function CatalogBookHandler() {
 	 * @param $roleAssignments array
 	 */
 	function authorize($request, &$args, $roleAssignments) {
-		import('classes.security.authorization.OmpPublishedMonographAccessPolicy');
-		$this->addPolicy(new OmpPublishedMonographAccessPolicy($request, $args, $roleAssignments));
+		import('lib.pkp.classes.security.authorization.ContextRequiredPolicy');
```

Comment:
I think this might skip some important checks. Usually the `...RequiredPolicy` says that the object must be fetchable based on a provided ID, then `...AccessPolicy` builds on that to ensure that the user has permission to view it. Swapping out `OmpPublishedMonographAccessPolicy` for `OmpPublishedMonographRequiredPolicy` would miss the access checks, no?